### PR TITLE
added gold NSC, updated RBA

### PR DESCRIPTION
**Pull request by stocchi**
gold NSC: new route, i used my notes but i tried to rearrange them as a guide, can be improved
blue RBA: kingler>charizard, new step by step cooltrainer explanation and more, missing: set animations on after giovanni

### CODE REVIEW

File: docs/gen-1/red-blue/catext/reverse-badge-acquisition/README.md [hunk 1]
```diff
@@ -52,20 +53,37 @@ A detailed guide on what strategy could be best for you and how to execute it ca
 - Get CoolTrainer glitch:
   - encounter any Rattata or a lv5 Pidgey
   - in the encounter, use trasform, then swap your moves order and run away
-- Catch missingno. (1 tile left from NSC)
+- Catch first missingno:
+  1. Open the Pokedex [here](https://www.gunnermaniac.com/pokeworld?map=1#64/221)
+  2. Look for an encounter
+  3. Enter the fight menu, scroll down until the move's type change
```
Comment (TiKevin83) [inline docs/gen-1/red-blue/catext/reverse-badge-acquisition/README.md]: change->changes

File: docs/gen-1/red-blue/catext/reverse-badge-acquisition/README.md [hunk 2]
```diff
@@ -52,20 +53,37 @@ A detailed guide on what strategy could be best for you and how to execute it ca
...
+  1. Toss all your items but Paralyz Heals
+  2. Throw all but 2 Paralyz heals
+  3. Swap slot 1 with slot2 twice
 - Swap slot 2 with ID balls (scroll down until you see jx[3], jx[0, **Masterball**])
 - Toss 1 from options 2 above
-- Catch Charizard (_TODO_ add tile to CoolTrainer on)
+- Catch Kingler:
+  1. Open the Pokedex [here](https://www.gunnermaniac.com/pokeworld?map=1#65/221)
+  2. Look for an encounter
+  3. Enter the fight menu, scroll down until the move's type change
```
Comment (TiKevin83) [inline docs/gen-1/red-blue/catext/reverse-badge-acquisition/README.md]: same here change->changes

File: docs/gen-2/gold-silver/catext/nsc/README.md [hunk 3]
```diff
@@ -0,0 +1,257 @@
...
+ <img src="https://cdn.discordapp.com/attachments/638862200148066324/874377359611031622/don.png"/>
+ </details>
+
+Avoid Spinner Don (rage both Caterpies if you hit)
+
+Optional: Pick up the Bitterberry and equip it when you heal in Falkner's Gym (Would recommend if you aren’t doing Don Manip)
+
+Violet Town:
+
+Enter Falkners gym
```
Comment (TiKevin83) [inline docs/gen-2/gold-silver/catext/nsc/README.md]: Falkners->Falkner's

File: docs/gen-2/gold-silver/catext/nsc/README.md [hunk 4]
```diff
@@ -0,0 +1,257 @@
...
+Equip Berry to Toto, Swap Toto with kenya
+walk left 1, up 1, left 2
+Open pokedex, scroll with right until bellsprout, open summary to hear its cry
+use coin case
+walk down, left, up
+talk to red (press A)
+
+
+Notes for coin case setup:
+- Croconaw always needs to be 4th in part and its movesets needs to be Scratch/Leer/FuryCutter/WaterGun, in this specific order
```
Comment (TiKevin83) [inline docs/gen-2/gold-silver/catext/nsc/README.md]: part->party

File: docs/gen-2/gold-silver/catext/nsc/README.md [hunk 5]
```diff
@@ -0,0 +1,257 @@
...
+- Either Egg or Kenya in 3rd is ~71% to work with random DVs (L6 Bellsprout is 71%*(12/16) to work, L3 Bellsprout never works
+- TODO: check sandshrew chances
+
+If you want to finish the run save after getting coin case, and if the glitch fails try different pokemons in slot3.
```
Comment (TiKevin83) [inline docs/gen-2/gold-silver/catext/nsc/README.md]: suggesting: If you want to finish the run, save after getting coin case. If the glitch fails try different pokemons in slot3.

File: docs/gen-2/gold-silver/catext/nsc/README.md [hunk 6]
```diff
@@ -0,0 +1,257 @@
...
+Rocket Grunt #2:
+- Zubat: Water Gun 2x
+  - Scratch if already L15
+- Ekans: Scratch 3x
+
+Heal to 10+ HP
+
+Rocket Grunt #3:
+- Rattata: Spam Rage
+  - if you don’t get a Rage stack on Rat Water Gun Zubats
```
Comment (TiKevin83) [inline docs/gen-2/gold-silver/catext/nsc/README.md]: comma after Rat

**Review by TiKevin83 — CHANGES_REQUESTED** [review]

**Comment by stocchi** [comment]
i fixed everything (hopefully), thanks for the review

**Review by TiKevin83 — APPROVED** [review]

**Pull request merged**